docs/feature_ps2_mouse.md

```diff
@@ -88,6 +88,32 @@ In your keyboard config.h:
 #endif
 ```
 
+### Interrupt Version (ARM chibios)
+
+Pretty much any two pins can be used for the (software) interrupt variant on ARM cores. The example below uses A8 for clock, and A9 for data.
+
+In rules.mk:
+
+```
+PS2_MOUSE_ENABLE = yes
+PS2_USE_INT = yes
+```
+
+In your keyboard config.h:
+
+```c
+#define PS2_CLOCK A8
+#define PS2_DATA  A9
+#define PS2_CLOCK_PORT EXT_MODE_GPIOA
```

Comment:
You can use something like 
`#define ExtModePort(pin) (((uint32_t)PAL_PORT(pin) & 0x0000FF00U) >> 6)`
and then `ExtModePort(PS2_DATA)` instead of having a config option to work out EXT_MODE_GPIOA

Reply:
that's indeed a bit nicer - from a enduser perspective who's having to configure stuff